source/index.js

```diff
@@ -76,7 +78,7 @@ const buildResult = (options, parserOptions) => {
 		input,
 		flags,
 		unnormalizedFlags,
-		pkg: package_,
+		pkg: packageJson,
```

Comment:
Thoughts on renaming `cli.pkg` to `package` or `packageJson`?

Reply:
It cannot be `package` as it's a reserved keyword, so it cannot be destructured.

Reply:
Hadn't considered the destructuring part, fair point. I'm ambivalent on the change, but feel free to add it to #233 if you'd like to change it (or just go for it). The consistency (and not having to rename variables) would be nice though.